src/pages/SignUpPage.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import '../style/App.scss';
+import SignUpForm from '../components/users/SignUpForm';
+import AppStoreContainer from '../components/app-store-container/AppStoreContainer';
+
```

Comment:
```suggestion
const SignUpPage = () =>
    <div className="main-div">
      <div className="left-container">
        <p className="left-container__title">
          <FormattedMessage id="loginform.signup.text" />
        </p>
        <SignUpForm />
      </div>
      <AppStoreContainer />
    </div>
```

Reply:
Linter doesn't allow me to do this.

Reply:
I would remove that rule, I find far more readable this way. At least the one with the round brackets. 
```suggestion
const SignUpPage = () => (
    <div className="main-div">
       ...
    </div>
   );
```

Reply:
it worked 👌  with parenthesis